tests/AzureBlobStorageAdapterTest.php

```diff
@@ -0,0 +1,480 @@
+    ////                                    "Content-Length" => "4445",
+    ////                                    "Content-Type" => "image/png",
+    ////                                    "Content-MD5" => "YDE4L4f/zE+XvOicUOGk0g==",
+    ////                                    "BlobType" => "BlockBlob",
+    ////                                    "LeaseStatus" => "unlocked",
+    ////                                    "LeaseState" => "available",
+    ////                                    "ServerEncrypted" => "true",
+    ////                                    "AccessTier" => "Hot",
+    ////                                    "AccessTierInferred" => "true",
+    ////                                    "AccessTierChangeTime" => "Thu, 16 Dec 2021 17:42:12 GMT",
+    ////                                ],
+    ////                            ],
+    ////                            [
+    ////                                "Name" => "test_image.png",
+    ////                                "Properties" => [
+    ////                                    "Creation-Time" => "Thu, 16 Dec 2021 17:42:12 GMT",
+    ////                                    "Last-Modified" => "Thu, 16 Dec 2021 17:42:12 GMT",
+    ////                                    "Etag" => "0x1B6041BF1747F80",
+    ////                                    "Content-Length" => "4445",
+    ////                                    "Content-Type" => "image/png",
+    ////                                    "Content-MD5" => "YDE4L4f/zE+XvOicUOGk0g==",
+    ////                                    "BlobType" => "BlockBlob",
+    ////                                    "LeaseStatus" => "unlocked",
+    ////                                    "LeaseState" => "available",
+    ////                                    "ServerEncrypted" => "true",
+    ////                                    "AccessTier" => "Hot",
+    ////                                    "AccessTierInferred" => "true",
+    ////                                    "AccessTierChangeTime" => "Thu, 16 Dec 2021 17:42:12 GMT",
+    ////                                ],
+    ////                            ],
+    ////                        ],
+    ////                    ],
+    ////                    "NextMarker" => null,
+    ////                ]
+    ////            ]));
+    ////        $blobClient = BlobRestProxy::createBlobService('AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://azurite:10000/devstoreaccount1;QueueEndpoint=http://azurite:10001/devstoreaccount1;TableEndpoint=http://azurite:10002/devstoreaccount1;');
+    //        $service = new AzureBlobStorageAdapter($blobClient, $this->logger, 'default');
+    //        $response = $service->listContents('', true);
+    //        $this->assertIsIterable($response, 'Error getting the list of the content');
+    //    }
```

Comment:
Is this really needed?

Reply:
for now, yes, to allow us, continue more easy the test, now we have like 80% of coverage